mm2src/coins/eth.rs

```diff
@@ -3826,9 +3875,9 @@ impl EthCoin {
                     U256::from(self.gas_limit.eth_send_erc20),
                 )
             },
-            EthCoinType::Nft { .. } => Box::new(futures01::future::err(TransactionErr::ProtocolNotSupported(ERRL!(
-                "Nft Protocol is not supported yet!"
-            )))),
+            EthCoinType::Nft { .. } | EthCoinType::Trx => Box::new(futures01::future::err(
+                TransactionErr::ProtocolNotSupported(ERRL!("Nft and Trx Protocols are not supported yet!")),
+            )),
```

Comment:
why not specify whether it's NFT or TRX like the other cases?

other similar occurrences in this file as well.

Reply:
thanks for finding, i missed it, https://github.com/KomodoPlatform/komodo-defi-framework/pull/2467/commits/1f69c9d0e62b5e8f996f29f04c50a51a2e6baec8